pkg/utils/integration/test_scenarios.go

```diff
@@ -0,0 +1,129 @@
+package integration
```

Comment:
The resource we're dealing with here is called "IntegrationTestScenario", so can we rename this file to something like `integration_test_scenarios.go`?

Reply:
Sure. Will do.

Reply:
Renamed